Makefile

```diff
@@ -3,13 +3,49 @@
 SOURCES = AddressResolver.cpp Profile.cpp
 HEADERS = AddressResolver.h Profile.h
 
+KCACHEGRIND = kcachegrind
+
+CFLAGS = -Wall -Wextra -Werror -g -fno-omit-frame-pointer
```

Comment:
-Werror is overkill, new compiler can add a new warning, we don't want to stop the build because of that.
-fno-omit-frame-pointer is not needed for perfgrind, but for profiled application

We can add -O2 though :)

Reply:
I need something to check later, so I use pginfo for that. Maybe only add it there and have `-O2` in the general flags?

Reply:
pginfo uses same Profile.o and AddressResolver.o objects...

Reply:
That would be possible - but we don't create those separately and instead let GCC compile them each time and statically link them into the executables directy.

... I'll add another pginfo_dbg target which is then only used for the checks.